firmware/src/sgx/src/untrusted/keyvalue_store.c

```diff
@@ -23,26 +23,54 @@
  */
 
 #include <sys/stat.h>
+#include <ctype.h>
 #include "hsm_u.h"
 #include "log.h"
 
 #define KVSTORE_PREFIX "./kvstore-"
 #define KVSTORE_SUFFIX ".dat"
+#define KVSTORE_MAX_KEY_LEN 256
```

Comment:
This doesn't take into account the prefix and suffix. I wouldn't push it to the maximum, keep it at a safe enough size (like 150 or something).

Reply:
Sounds good, I'll do that change, thanks!